sass/_chatbox.scss

```diff
@@ -259,16 +259,12 @@
             }
 
             .chat-textarea {
-                color: var(--chat-textarea-color);
-                border-top-left-radius: 0;
-                border-top-right-radius: 0;
-                border-bottom-radius: var(--chatbox-border-radius);
-                padding: 0.5em;
-                width: 100%;
-                border: none;
-                min-height: var(--chat-textarea-height);
-                margin-bottom: -4px; // Not clear why this is necessar :(
-                resize: none;
+		     color: var(--chat-textarea-color);
+		     border: 1px solid #fff;
+                     margin: 0 0 0 0px;
+                     min-height: 40px;
+                     padding: 10px 75px 10px 10px;
```

Comment:
Ideally responsive values (e.g. `0.5em`) should not be replaced with fixed values.

Why was this change necessary?